package.json

```diff
@@ -33,7 +33,7 @@
     "url-search-params": "^0.6.1"
   },
   "scripts": {
-    "_format": "prettier-eslint --write \"src/**/*.js\"",
+    "_format": "prettier-eslint --write \"src/**/*.js\" \"src/**/*.jsx\"",
```

Comment:
Do we have any .jsx files?
I wasn't planning to use that file extension in this project.